packages/frontend/src/components/RichTextEditor/index.tsx

```diff
@@ -0,0 +1,130 @@
+import './RichTextEditor.scss'
+
+import { useEffect } from 'react'
+import { Controller, useFormContext } from 'react-hook-form'
+import { FormControl } from '@mui/material'
+import { FormLabel } from '@opengovsg/design-system-react'
+import Link from '@tiptap/extension-link'
+import Placeholder from '@tiptap/extension-placeholder'
+import Table from '@tiptap/extension-table'
+import TableCell from '@tiptap/extension-table-cell'
+import TableHeader from '@tiptap/extension-table-header'
+import TableRow from '@tiptap/extension-table-row'
+import Underline from '@tiptap/extension-underline'
+import { EditorContent, useEditor } from '@tiptap/react'
+import StarterKit from '@tiptap/starter-kit'
+
+import { MenuBar } from './MenuBar'
+
+const Editor = ({
+  onChange,
+  initialValue,
+  editable,
+  placeholder,
+}: {
+  onChange: (...event: any[]) => void
+  initialValue: string
+  editable?: boolean
+  placeholder?: string
+}) => {
+  const editor = useEditor({
+    extensions: [
+      StarterKit,
+      Link.configure({
+        HTMLAttributes: { rel: null, target: '_blank' },
+      }),
+      Underline,
+      Table.configure({
+        resizable: false,
+        HTMLAttributes: {
+          style: 'border-collapse:collapse;',
+        },
+      }),
+      TableRow,
+      TableHeader,
+      TableCell.configure({
+        HTMLAttributes: {
+          style: 'border:1px solid black;',
+        },
+      }),
+      Placeholder.configure({
+        placeholder,
+      }),
+    ],
+    content: initialValue,
```

Comment:
It looks like prosemirror is stripping out unknown tags. E.g. create a pipe with something like :
```
<p>first para</p>
<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/2/2f/Google_2015_logo.svg/272px-Google_2015_logo.svg.png" />
<p>2nd para</p>
```
The image is stripped out, and data is lost if user tries to edit the step.

Is there a way we can configure PM to display unknown HTML as raw Text

Reply:
I think bopian. I thought our users dont need images but just did a look up in our db they actually need image support. Will be adding it